src/lib/components/lines-and-dots/svg/timeline-graph-row.svelte

```diff
@@ -37,6 +47,29 @@
   $: pendingActivity = group?.pendingActivity;
   $: pauseTime = pendingActivity && pendingActivity.pauseInfo?.pauseTime;
 
+  let decodedLocalActivity: SummaryAttribute | undefined;
+
+  onMount(async () => {
+    if (hasLocalActivityMarker(group)) {
+      const localActivityEvent = getLocalActivityMarkerEvent(group);
```

Comment:
Is there a reason for both `hasLocalActivityMarker` and `getLocalActivityMarkerEvent`? Seems like we could just determine if there is a local activity marker event with `getLocalActivityMarkerEvent` (especially since there is already a `if (localActivityEvent)` condition below) instead of iterating over the group twice.